src/components/Content/Content.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './Content.scss';
+import { Tabs } from '../../statics/Statics';
+import Sidebar from '../Sidebar/Sidebar';
+import DataSection from '../DataSection/DataSection'
+
+function Content () {
+    const [activeTab, setActiveTab] = useState(Tabs[0].label)
+  
+  return (
+    <div className="Content__Container">
+      <Sidebar activeTab={activeTab} changeTab={tab => setActiveTab(tab)} />
```

Comment:
You can just pass in the reference to `setActiveTab` instead of creating a function which does that.

Reply:
Makes sense. Made changes.